ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -4028,6 +4030,8 @@ public static int getFooterCount(TableDesc table, JobConf job) throws IOExceptio
     int footerCount;
     try {
       footerCount = Integer.parseInt(table.getProperties().getProperty(serdeConstants.FOOTER_COUNT, "0"));
+      footerCount =
+          validateHeaderFooter(table, footerCount, "skip.footer.line.count");
```

Comment:
since `FOOTER_COUNT = "skip.footer.line.count"` ; I think you could also push in this `Integer.parseInt` into you method as well

Reply:
Done, Thanx